### PR TITLE
refine api key signup page

### LINKED ISSUE #4363: Incorporate design/UX and feature flag for API key sign up
**What we're after:**
The vanilla sign up key page prototype has been completed. There is outstanding feedback that needs to be addressed prior to merging the code.

**Completion criteria:**
- [x] The sign up key page is under a feature flag
- [x] The design/UX feedback is incorporated per https://github.com/fecgov/openFEC/pull/4359#issuecomment-630270654 (per @JonellaCulmer)
    - [x] We include help text that says to include information where "* Required", but none of the fields include an asterisk. Can you include one where appropriate? Or add more "(optional)" text everywhere else.
    - [x] We need to reduce the size of the header for the sign-up text. Currently, it's the same size and prominence as the page title. Can we reduct the text "Sign up for an API key" to 24px?
    - [x] Can we add a second sentence that explains that users can still use the demo key lower on the page? Not sure how to phrase it. Example: Signing up here does not prevent you from trying out the API below using a DEMO_KEY. For those interested in doing XXX, please sign up for an API key.
- [x] Incorporate finalized language for API key sign up page
- ~[ ] Incorporate finalized language for API key sign up email, also include api.open.fec.gov in the example within the email~moved to issue #4445 by @JonellaCulmer 
- [x] Remove `Get an [API key here](https://api.data.gov/signup/)` when the feature flag is turned on https://github.com/fecgov/openFEC/pull/4359#issuecomment-636054696
- [x] Clean up JS per @johnnyporkchops https://github.com/fecgov/openFEC/pull/4359#pullrequestreview-421086744 and
- [x] Clean up comments per @johnnyporkchops https://github.com/fecgov/openFEC/pull/4359#pullrequestreview-421087807
- [x] Add API_UMBRELLA_SIGNUP_KEY for each environment

### PR DESCRIPTION
**Pull request by jason-upchurch**
## Summary (required)

- Resolves #4363
- hard coding "required fields" addressed under #4447

refine API key signup elements per completion criteria in #4363 

## How to test the changes locally
for non developers: 
- https://fec-dev-api.app.cloud.gov/developers/

for developers:
- check out branch
- pytest
- run local api
- verify that api signup looks correct per #4363 completion criteria.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  swagger-ui.html
- docs.py

### CODE REVIEW

**Comment by jason-upchurch** [comment]
> Thanks for your work on this, @jason-upchurch! Before merge, can you remove the extra `dev` deploy?
> 
> https://github.com/fecgov/openFEC/blob/90097b72568990d5a65ecdd4c5aa3ad2e8900d6a/tasks.py#L145-L146

@lbeaufort we don't want to merge the dev deploy code in. That's there for the benefit of anyone wanting to review on dev. @JonellaCulmer @johnnyporkchops @pkfec can you please make a note in the issue if you do not plan to review? I will make one last amended commit before merge to remove the dev deploy (or redeploy to dev should any reviewers want to inspect my work on dev).

**Comment by JonellaCulmer** [comment]
@jason-upchurch I don't see any of the changes reflected in the https://fec-dev-api.app.cloud.gov/developers/ link you provided.

**Comment by jason-upchurch** [comment]
> @jason-upchurch I don't see any of the changes reflected in the https://fec-dev-api.app.cloud.gov/developers/ link you provided.

@JonellaCulmer that's because  dev has been overwritten many times  since I deployed this I think a week or so ago. I'll check with the team and redeploy it for you.

File: webservices/templates/swagger-ui.html [hunk 1]
```diff
@@ -23,9 +23,45 @@
...
+      margin:0;
+      background: #fafafa;
       }
+      
+      label[for=user_first_name]:before
+      {
+      content: " *";
+      border-style: none;
+      cursor: default;
+      color: #E87723;
```
Comment (johnnyporkchops) [inline webservices/templates/swagger-ui.html:35]: I don't think you  need `border-style` or `cursor`  for any of the `:before`s , unless you put them in for s specific reason I am missing

File: webservices/templates/swagger-ui.html [hunk 2]
```diff
@@ -23,9 +23,45 @@
 
       body
       {
-        margin:0;
-        background: #fafafa;
+      margin:0;
```
Comment (johnnyporkchops) [inline webservices/templates/swagger-ui.html:26]: Just a FLAK8 formatting issue: there should be a space before `0`

Comment (jason-upchurch) [reply webservices/templates/swagger-ui.html]: fixed!  Thanks @johnnyporkchops

File: webservices/templates/swagger-ui.html (hunk 1 above)
Comment (jason-upchurch) [reply webservices/templates/swagger-ui.html]: fixed!  Thanks @johnnyporkchops

**Pull request merged**